internal/character/bailu/technique.go

```diff
@@ -0,0 +1,15 @@
+package bailu
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/key"
+)
+
+// After Technique is used, at the start of the next battle,
+// all allies are granted Invigoration for 2 turn(s).
+
+func (c *char) Technique(target key.TargetID, state info.ActionState) {
+	for _, char := range c.engine.Characters() {
+		c.addInvigoration(char, 2)
+	}
```

Comment:
Duration of her technique's invigoration is affected by A4, so this needs to check for that